lib/css/components/_stacks-activity-indicator.less

```diff
@@ -13,9 +13,17 @@
 
 .s-activity-indicator {
     display: inline-block;
-    width: @su12;
+    text-align: center;
+    min-width: @su12;
     height: @su12;
+    padding-left: @su4;
+    padding-right: @su4;
+    line-height: 1;
     background-color: var(--theme-secondary-400);
     box-shadow: 0 0 0 @su4 var(--focus-ring);
-    border-radius: 100%;
+    border-radius: 1000px;
+    font-size: @fs-fine;
+    font-weight: 600;
+    color: @white;
```

Comment:
This should be `var(--white)` to play with nice with dark mode, correct?

as is:
![20210907-112954](https://user-images.githubusercontent.com/647177/132371712-627191fb-478c-43d4-90c9-b482e9ac876d.png)

using `var(--white)`:
![20210907-113005](https://user-images.githubusercontent.com/647177/132371770-3445dcb7-b8cf-4cfc-a862-66995a3f3c05.png)

```suggestion
    color: var(--white);
```

Reply:
I chose white to try to keep contrast :v: